src/components/utils/handleStorage.js

```diff
@@ -1,7 +1,30 @@
-export function saveToStorage(key, dataSet) {
-  localStorage.setItem(key, JSON.stringify(dataSet))
+import firebase from 'firebase'
```

Comment:
I wouldnt recommend a 'utils' folder. 
better is a 'services' folder ... utils is to generic and broad.